oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexHelperTest.java

```diff
@@ -145,4 +146,8 @@ public void spellCheck() throws IOException {
         assertThat(barFields.get("trigram"), nullValue());
     }
 
+    public static String getRemoteIndexName(ElasticIndexDefinition definition) {
+        return definition.getRemoteIndexAlias() + "-" + UUID.randomUUID().getMostSignificantBits();
```

Comment:
this application logic should not be in a test. If we change the logic we need to remember to change the test.